src/community/who-uses-dart.md

```diff
@@ -4,23 +4,88 @@ description: Mobile apps and Google web apps are two of the most common uses for
 toc: false
 ---
 
-The popular [Flutter toolkit][Flutter] relies on the Dart language,
-so developers writing Flutter apps—for Android, iOS, or other targets—use Dart.
-To find a list of Flutter apps, visit the [Flutter showcase.][showcase]
+## Flutter
 
-[Flutter]: {{site.flutter}}
-[showcase]: {{site.flutter}}/showcase
+The popular [Flutter toolkit][] is powered by the Dart language,
+runtimes, tooling, and libraries. Over 500,000 Flutter apps for Android,
+iOS, the web, and other targets, are all Dart-based.
```

Comment:
**issue:** Active voice, per the [GSG](https://developers.google.com/style/tone).
```suggestion
The Dart language and ecosystem powers the popular [Flutter toolkit][].
Developers have created more than 500,000 Flutter apps for Android,
iOS, the web, and other targets using Dart.
```